srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py

```diff
@@ -314,6 +314,17 @@ def main():
     worker_pool = pool.Pool(processes=n_processes)
     worker_pool.starmap(generate_fault_realisations, messages)
 
+    if args.aggregate_file is not None:
+        ordered_rels = [
+            get_realisation_name(pid, i + 1) for pid in pids for i in faults[pid]
+        ]
+
+        agg = pd.read_csv(args.aggregate_file)
+        agg = agg.sort_values(
```

Comment:
Consider using pandas categoricals to sort the df?

```df['LSE_cat'] = pd.Categorical(
    df['LSE'], 
    categories=['Oands','Wetnds','Develd','Cn','Soys','Otherg','Wht'], 
    ordered=True
)
df.sort_values('LSE_cat')```

Reply:
I considered it, but that involves a lot more faffing around to achieve the same result